core/state_transition.go

```diff
@@ -262,6 +262,22 @@ func (st *StateTransition) buyGas(gasBailout bool) error {
 	if overflow {
 		return fmt.Errorf("%w: address %v", ErrInsufficientFunds, st.msg.From().Hex())
 	}
+
+	// compute data fee for eip-4844 data blobs if any
+	dgbig := new(big.Int)
```

Comment:
nit, `s/dgbig/dgBig` and `s/dgval/dgVal`, camelCase?

Reply:
the code here already uses "mgval" (no camel case) so I was torn between following the local code style or the go idiom of camel casing.   let me at least fix "dgbig" since it's a new variable.

Reply:
saw the `mgval` part, it's a nit anyway, no worries